static/js/pages/instances/InstanceDetailPanel.tsx

```diff
@@ -139,46 +235,61 @@ const InstanceDetailPanel: FC<Props> = ({ instance, notify, onClose }) => {
             </Link>
           ))}
         />
-        <h5 className="u-no-margin--bottom">Networks</h5>
+        <hr className="u-spaced-hr" />
+        <h5 className="p-text--x-small-capitalised u-no-margin--bottom">
+          Networks
+        </h5>
         <List
           className="u-no-margin--bottom"
           items={Object.values(instance.expanded_devices)
             .filter(isNicDevice)
-            .map((item) =>
-              item.name ? `${item.name}: ${item.network}` : item.network
-            )}
+            .map((item) => (
+              // TODO: fix this link to point to the network detail page
+              <Link key={item.network} to={`/ui/${instance.project}/networks`}>
+                {item.network}
+              </Link>
+            ))}
         />
+        <hr className="u-spaced-hr" />
         <Row>
-          <Col size={1}>
-            <h5 className="u-no-margin--bottom">Snapshots</h5>
+          <Col size={3}>
+            <h5 className="p-text--x-small-capitalised u-no-margin--bottom">
+              Recent Snapshots
+            </h5>
           </Col>
-          <Col
-            size={3}
-            className="p-snapshot-link p-text--small u-align--right u-no-margin--bottom"
-          >
-            <Link
-              to={`/ui/${instance.project}/instances/${instance.name}/snapshots`}
+          {instance.snapshots?.length && (
+            <Col
+              size={1}
+              className="p-snapshot-link p-text--small u-align--right u-no-margin--bottom u-truncate"
             >
-              Manage snapshots
-            </Link>
-          </Col>
+              <strong>
+                <Link
+                  to={`/ui/${instance.project}/instances/${instance.name}/snapshots`}
```

Comment:
This link should always be visible, also with no snapshots available. Otherwise, it is hard to create your first snapshot.

Reply:
This needs design input because showing a link that says "View all" when you have "None" written just below seems counterintuitive to me.
Please discuss this with design and once clarified I'll be happy to update.

Reply:
I discussed with Alex, and we agreed this will likely change as soon as we start working on snapshots in a couple of weeks.
For now, we can change the label to from "View all" to "Manage", when all items in this view are visible (i.e. no items are hidden).

Reply:
Sounds good, I'm not sure I understood correctly the part about "all items in this view are visible" - can they even be hidden? By the user? Apart from that, I changed the label and removed the check that was hiding it when no snapshot was present.

Reply:
What I meant is:
We limit the number of displayed snapshots in the side panel. If there are more available, the label should say "view all", otherwise "manage"